home-data-migration/src/main/resources/db/changelog/v-1.0/001-create-users-table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE if not exists public.users (
+                            id int not null  primary key,
```

Comment:
Please revisit the data type. `int` -> `bigint`

Reply:
Fixed.